locales/en/translations.json

```diff
@@ -1,12 +1,18 @@
 {
     "addPartnerLegalEntityPage" : {
         "generalPartner": {
-            "title" : "Add a general partner (legal entity)"
+            "title" : "Add a general partner (legal entity)",
+            "dateEffectiveFrom" : "When did they become a general partner for this partnership?"
         },
         "limitedPartner": {
-            "title": "Add a limited partner (legal entity)"
+            "title": "Add a limited partner (legal entity)",
+            "dateEffectiveFrom" : "When did they become a limited partner for this partnership?"
         },
 
+        "dateHint" : "For example, 27 3 2000",
```

Comment:
Date hint example date doesn't match that which is on the ticket screen-shot, but maybe this is ok.